MapboxNavigation/NavigationMapView.swift

```diff
@@ -852,6 +931,8 @@ public protocol NavigationMapViewDelegate: class  {
     
     @objc optional func navigationMapView(_ mapView: NavigationMapView, routeCasingStyleLayerWithIdentifier identifier: String, source: MGLSource) -> MGLStyleLayer?
     
+    @objc optional func navigationMapView(_ mapView: NavigationMapView, didTap routeIndex: Int, routes: [Route])
```

Comment:
This method should return the Route itself, not its index in the list of routes.